test/comparison-tests/create-and-execute-test.js

```diff
@@ -371,6 +371,8 @@ function getNormalisedFileContent(file, location, test) {
                 .replace(/[\d]+[.][\d]* kB/g, ' A-NUMBER-OF kB')
                 // We also don't want a difference in the number of bytes to fail the build
                 .replace(/ \d+ bytes /g, ' A-NUMBER-OF bytes ')
+                // Sometimes "[built]" is written to output, and sometimes not. This should not fail the build
```

Comment:
Well, in the previous PR, and in the first integration test of this PR, I saw that one test failed, but only for Node 4, because `[build]` was not there. However the test succeeded for all other versions of Node and TypeScript.